src/modules/add.js

```diff
@@ -14,10 +14,20 @@ const add = (e) => {
     section.insertAdjacentElement('afterend', message);
     setTimeout(() => { message.remove(); }, 2000);
   } else {
-    const score = new Scores(name, scoreV);
+    // const score = new Scores(name, scoreV);
 
-    Interface.addScoreToList(score);
-    Store.addScore(score);
+    // Interface.addScoreToList(score);
+    // Store.addScore(score);
```

Comment:
- [ ] to keep your repo clean and professional, kindly remove any commented code in your project 🙏